client/src/components/ProjectForm.js

```diff
@@ -72,6 +73,21 @@ export default function ProjectForm({
   // State to track the toggling from Project view to Edit Project View via edit icon.
   const [editMode, setEditMode] = useState(false);
   const { auth } = useAuth();
+  const [isModalOpen, setIsModalOpen] = useState(false);
+  const handleOpen = () => setIsModalOpen(true)
+  const handleClose = () => setIsModalOpen(false)
+
+  const style = {
```

Comment:
This `style` object is defined in `ProjectForm.js` but not used. It's duplicated in `ChangesModal.js`

Reply:
@Spiteless  fixed